frontend/common.js

```diff
@@ -134,6 +141,32 @@ export default {
           }
           Vue.set(this.indiWorld[deviceName], propertyName, state);
         },
+        fetchRoutes() {
+          this.$axios.get(constants.CONFIG_URL)
```

Comment:
I think we can skip adding axios and just use the fetch api directly here

Reply:
here's an example: https://github.com/magao-x/sup/blob/main/frontend/components/plots/ObservabilityPlots.vue#L97

Reply:
Sure, that makes sense, I've refactored to use fetch instead. I've also moved the call in index.js so that the relevant routes are loaded from the beginning and changed to switched to storing config data in store which removes the need to inject it in elements.